packages/web/src/components/gcds-fieldset/gcds-fieldset.css

```diff
@@ -30,6 +34,12 @@
 
       .legend__required {
         margin: var(--gcds-fieldset-legend-required-margin) !important;
+        font: var(--gcds-text-size-body-desktop); /* --gcds-fieldset-legend-required-font-desktop */
+        vertical-align: middle;
+
+        @media only screen and (width < 48em) {
+          font: var(--gcds-text-size-body-mobile);
```

Comment:
Same comment here, this change seems redundant if we only need to change the font-weight. Ignore this if we also need to change other font properties to different values than the legend font values.

Reply:
I was originally only going to update font weight but when viewing the design in figma I noticed the (required) label in fieldset uses body font settings. Should be updated to the correct new token now.